shared/locales/en/feedback.json

```diff
@@ -0,0 +1,54 @@
+{
+    "smartling": {
+        "string_format" : "icu",
+        "translate_paths": [{
+            "path": "*/title",
+            "key": "{*}/title",
+            "instruction": "*/note"
+        }]
+    },
+    "brokenSiteLabel": {
+        "title": "Which website is broken?",
+        "note": "Input label for broken site URL"
+    },
+    "brokenSitePlaceholder": {
+        "title": "Copy and paste your URL",
+        "note": "Placeholder text on URL input"
```

Comment:
"Instructions to fill an input field with a URL. Here "your URL" doesn't mean that they own the website, just that they're reporting it."